oda_api/data_products.py

```diff
@@ -765,6 +765,7 @@ class LightCurveDataProduct(NumpyDataProduct):
     @classmethod
     def from_arrays(cls,
                     times, 
+                    timedel = None,
```

Comment:
This finally isn't used

Reply:
I have implemented it and modified the relative test.